### PR TITLE
Codestyle installation views

### PR DESCRIPTION
**Pull request by zero-24**
#### Summary of Changes

Smal CodeStyle in the install folder
#### Testing Instructions

Review the code or make sure you can install: https://github.com/zero-24/joomla-cms/archive/installcodestyle.zip

### CODE REVIEW

File: installation/view/install/tmpl/default.php [hunk 1]
```diff
@@ -10,14 +10,12 @@
 
 /* @var InstallationViewInstallHtml $this */
 ?>
-<form action="index.php" method="post" id="adminForm" class="form-validate form-horizontal x">
+<form action="index.php" method="post" id="adminForm" class="form-validate form-horizontal">
```
Comment (andrepereiradasilva) [inline installation/view/install/tmpl/default.php]: this x class is not needed?
the rest is ok

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 0b907202f65555ddbb75cdc489ed697fd3b98f7e<br /><br />Tested installation from the zip<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10359">issues.joomla.org/joomla-cms/10359</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 0b907202f65555ddbb75cdc489ed697fd3b98f7e<br /><br />i've used the zip for a successul installation<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10359">issues.joomla.org/joomla-cms/10359</a>.</sub>

File: installation/view/install/tmpl/default.php (hunk 1 above)
Comment (zero-24) [inline installation/view/install/tmpl/default.php]: There is no css definition for x ;) see: https://github.com/joomla/joomla-cms/tree/staging/installation/template/css

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10359">issues.joomla.org/joomla-cms/10359</a>.</sub>

File: installation/view/install/tmpl/default.php (hunk 1 above)
Comment (andrepereiradasilva) [inline installation/view/install/tmpl/default.php]: ok then

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 0b907202f65555ddbb75cdc489ed697fd3b98f7e<br /><br />on code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10359">issues.joomla.org/joomla-cms/10359</a>.</sub>

**Pull request merged**